roles/openshift_cloud_provider/templates/openstack.conf.j2

```diff
@@ -22,7 +22,8 @@ ca-file = {{ openshift_openstack_ca_file_path }}
 [LoadBalancer]
 subnet-id = {{ openshift_cloudprovider_openstack_lb_subnet_id }}
 {% endif %}
-{% if openshift_cloudprovider_openstack_blockstorage_version is defined %}
 [BlockStorage]
+ignore-volume-az = {{ openshift_cloudprovider_openstack_blockstorage_ignore_volume_az | ternary('yes', 'no') }}
```

Comment:
openshift_cloudprovider_openstack_blockstorage_ignore_volume_az should filtered to bool before passing to ternary.  Any user-supplied booleans must be filtered to bool before utilizing.

Reply:
Done! Thanks for the review!